Tests/Runner/RunnerTest.php

```diff
@@ -495,7 +536,6 @@ public function testExecutorStoreState()
         $configuration->setComponentId('keboola.docker-demo-sync');
         $configuration->setName('Test configuration');
         $configuration->setConfigurationId('runner-configuration');
-        $configuration->setState(json_encode(['foo' => 'bar']));
```

Comment:
no to bych tu nechal - testuje to, ze ta puvodni hodnota zmizi (ze se to treba nemerguje)

Reply:
no a nemělo by to teda bejt i v `$this->prepareJobDefinitions`, kde to nebylo? nedává smysl, že máš state v konfiguraci, ale ne v definici jobu. to testuje leda tak nějaká race conditions.

Reply:
testujes jestli jak se zmenil stav konfigurace, ne, co bylo uvnitr jobu

Reply:
tak to se teda testuje víc věcí naráz. udělám na to separátní test.

Reply:
jakto? ten test ma assertions jenom na stav konfigurace po dobehnuti kompoenenty, nic jinyho se v nem netestuje, nez se ulozil state

Reply:
no, kromě toho, že se uloží, ještě říkáš, že se testuje, že se state přepíše a nemerguje. to bych od sebe oddělil.

Reply:
ung, proc?

Reply:
tak jestli je to takovej problem, tak tam daj `assertCount(keys($configuration))` a splni to stejnej ucel

Reply:
jojo, to tam přidávám, jen bych to rozhodil do víc testů, ať je jasný, co se testuje?